styles/whitespace-colors-css.tmpl

```diff
@@ -7,8 +7,8 @@
 .CodeMirror .cm-dk-whitespace-tab:before { background-color: <%= light.whitespace %>; }
 .CodeMirror .cm-dk-whitespace-leading-space:before,
 .CodeMirror .cm-dk-whitespace-leading-tab:before { background-color: <%= light.leading %>; }
-.CodeMirror .cm-dk-whitespace-empty-line-tab:before,
-.CodeMirror .cm-dk-whitespace-empty-line-space:before { background-color: <%= light.empty %>; }
+.CodeMirror .cm-dk-whitespace-empty-line-space:before,
+.CodeMirror .cm-dk-whitespace-empty-line-tab:before { background-color: <%= light.empty %>; }
```

Comment:
These were simply in the wrong order. The rules are written in space-tab order, but this and the dark version was in tab-space order.